desktop/webpack.config.development.js

```diff
@@ -73,5 +84,11 @@ if (getenv.boolish('HOT', false)) {
     }
   })
 }
+
+if (USING_DLL) {
+  // Don't build the main thing
```

Comment:
This comment is a bit terse. Could you please expand upon it to clarify why we're not building it?